src/App/App.test.js

```diff
@@ -1,8 +1,23 @@
-import { render, screen } from '@testing-library/react';
+import { render, screen, fireEvent } from '@testing-library/react';
+import '@testing-library/jest-dom';
 import App from './App';
 
-test('renders learn react link', () => {
-  render(<App />);
-  const linkElement = screen.getByText(/learn react/i);
-  expect(linkElement).toBeInTheDocument();
+describe('App', () => {
+  it('should take user to movie cards view when return to home button is clicked', () => {
```

Comment:
Definitely want to see if we are doing this test in right test file. When we run our tests we'll find out!